src/clip.sol

```diff
@@ -162,6 +162,9 @@ contract Clipper is LibNote {
     function min(uint256 x, uint256 y) internal pure returns (uint256 z) {
         return x <= y ? x : y;
     }
+    function max(uint256 x, uint256 y) internal pure returns (uint256 z) {
```

Comment:
This function gets shadowed (solidity warning) by another usage of the word `max` in `take` function. I would rename the argument of the function.